python/setup.py

```diff
@@ -16,14 +16,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import print_function
 import glob
 import os
 import sys
 from setuptools import setup
 from shutil import copyfile, copytree, rmtree
 
-if sys.version_info < (3, 6):
```

Comment:
Maybe convert this into a warning that the user is using a version of Python that Spark has deprecated support for?

Reply:
+1 for @nchammas 's comment.

Reply:
Sure, will add.

Reply:
Hm, seems like it's tricky to print out logs during pip installation (see https://github.com/pypa/pip/issues/2933). Let me just merge this one.